tests/validation/cattlevalidationtest/core/test_machine.py

```diff
@@ -0,0 +1,277 @@
+
+    name = random_str()
+    create_args = {"name": random_str(),
+                   "digitaloceanConfig": {"accessToken": access_key,
+                                          }
+                   }
+
+    expected_values = {"image": default_image_name,
+                       "region": default_region,
+                       "size": default_size
+                       }
+    try:
+        digital_ocean_machine_life_cycle(client, create_args, expected_values)
+    finally:
+        delete_host_in_digital_ocean(name)
+
+
+@if_machine_digocean
+def test_digital_ocean_machine_parallel(client):
+    create_args = {"name": random_str(),
+                   "digitaloceanConfig": {"accessToken": access_key,
+                                          "image": image_name,
+                                          "region": region,
+                                          "size": size
+                                          }
+                   }
+    machines = []
+    try:
+        # Create 2 Digital Ocean Machines in parallel
+        for n in range(0, 2):
+            name = random_str() + "-parallel-" + str(n)
+            create_args["name"] = name
+            machine = client.create_machine(**create_args)
+            machines.append(machine)
+
+        # Check if both the machine and host get to "active" state
+
+        for machine in machines:
+            machine = client.wait_success(machine, timeout=DEFAULT_TIMEOUT)
+            assert machine.state == 'active'
+            machine = wait_for_host(client, machine)
+            host = machine.hosts()[0]
+            assert host.state == 'active'
+
+        for machine in machines:
+            machine = client.wait_success(machine.remove())
+            assert machine.state == 'removed'
+
+            host = machine.hosts()[0]
+            assert host.state == 'removed'
+            wait_for_host_destroy_in_digital_ocean(
+                host.ipAddresses()[0].address)
+    finally:
+        for machine in machines:
+            delete_host_in_digital_ocean(machine.name)
+
+
+@if_machine_digocean
+def test_digital_ocean_machine_invalid_access_token(client):
+
+    name = random_str()
+    create_args = {"name": name,
+                   "digitaloceanConfig": {"accessToken": "1234abcdefg",
+                                          "image": image_name,
+                                          "region": region,
+                                          "size": size
+                                          }
+                   }
+    # Create a Digital Ocean Machine with invalid access token
+    machine = client.create_machine(**create_args)
+    machine = wait_for_condition(client,
+                                 machine,
+                                 lambda x: x.state == 'error',
+                                 lambda x: 'Machine state is ' + x.state
+                                 )
+    assert error_msg_auth_failure in machine.transitioningMessage
```

Comment:
You changed this to use `in`, but I think you should also change `error_msg_auth_failure` to be simply `error_msg_auth_failure='401'`.

That will make it so that we are only looking for **401**, which should be less brittle.